base/shell/explorer/startup.cpp

```diff
@@ -332,3 +385,85 @@ ProcessStartupItems(VOID)
 
     return res ? 0 : 101;
 }
+
+static BOOL IsSelfExplorer(VOID)
+{
+    WCHAR szPath[MAX_PATH];
+
+    GetModuleFileName(NULL, szPath, _countof(szPath));
+    PathRemoveArgsW(szPath);
```

Comment:
This call is not needed because GetModuleFileName just retrieve the full path to the current executable.
It does not return any arguments that might have been passed in the command-line.

Reply:
OK. Cleanup'ed.